community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -176,7 +180,8 @@ public RelationshipCursor relationshipCursorGetAll( KernelStatement statement )
     public NodeCursor nodeCursorGetForLabel( KernelStatement statement, int labelId )
     {
         // TODO Filter this properly
-        return statement.getStoreStatement().acquireIteratorNodeCursor( storeLayer.nodesGetForLabel( statement, labelId ) );
+        return statement.getStoreStatement().acquireIteratorNodeCursor(
```

Comment:
yeah, I think longterm we will shift everything over to use cursors instead of iterators. No rush though.